vpr/src/draw/draw_debug.h

```diff
@@ -0,0 +1,35 @@
+/*This file contains all functions reagrding the graphics of setting and handling breakpoints*/
```

Comment:
typo (regarding)
Use Doxygen /** so we can build doxygen in future.
I'd say "related to the setting of place and route breakpoints"

Reply:
resolved